packages/@lightningjs/ui-components/src/components/Marquee/Marquee.js

```diff
@@ -267,7 +267,15 @@ export default class Marquee extends Base {
   }
 
   get textContent() {
-    return this.title && this.title.text ? this.title.text : this.title;
```

Comment:
instead of adding nested type checks, we can do this with just an `||`. we could also probably use a `?.` here instead of the ternary

Reply:
How about something like this: return this.title?.text ?? this.title ?? '';

Reply:
You could probably even just do `this.title?.text || this.title` unless you absolutely need the empty string and not undefined/null.